tools/onnx2tensorrt.py

```diff
@@ -0,0 +1,246 @@
+import argparse
+import os
+import os.path as osp
+from typing import Iterable, Optional, Union
+
+import mmcv
+import cv2
+import numpy as np
+import onnxruntime as ort
+import torch
+from mmcv.ops import get_onnxruntime_op_path
+from mmcv.tensorrt import (TRTWraper, is_tensorrt_plugin_loaded, onnx2trt,
+                           save_trt_engine)
+
+from mmedit.datasets.pipelines import Compose
+
+
+def get_GiB(x: int):
+    """return x GiB."""
+    return x * (1 << 30)
+
+
+def _prepare_input_img(model_type: str,
+                       img_path: str,
+                       config: dict,
+                       rescale_shape: Optional[Iterable] = None) -> dict:
+    """Prepare the input image
+
+    Args:
+        model_type (str): which kind of model config belong to, one of ['inpainting', 'mattor', 'restorer', 'synthesizer'].
+        img_path (str): image path to show or verify.
+        config (dict): MMCV config, determined by the inpupt config file.
+        rescale_shape (Optional[Iterable]): to rescale the shape of the input tensor.
+    
+    Returns:
+        dict: {'imgs': imgs, 'img_metas': img_metas}
+    """
+    # remove alpha from test_pipeline
+    model_type = model_type
+    if model_type == 'mattor':
+        keys_to_remove = ['alpha', 'ori_alpha']
+    elif model_type == 'restorer':
+        keys_to_remove = ['gt', 'gt_path']
+    for key in keys_to_remove:
+        for pipeline in list(config.test_pipeline):
+            if 'key' in pipeline and key == pipeline['key']:
+                config.test_pipeline.remove(pipeline)
+            if 'keys' in pipeline and key in pipeline['keys']:
+                pipeline['keys'].remove(key)
+                if len(pipeline['keys']) == 0:
+                    config.test_pipeline.remove(pipeline)
+            if 'meta_keys' in pipeline and key in pipeline['meta_keys']:
+                pipeline['meta_keys'].remove(key)
+
+    # build the data pipeline
+    test_pipeline = Compose(config.test_pipeline)
+    # prepare data
+    # if model_type == 'mattor':
```

Comment:
Remove these two commented lines if there are unnecessary.

Reply:
It is for matting usage, will be used in a short time I guess.

Reply:
If we do not support `mattor`, we could uncomented these two lines and raise error message inside if block.